src/web-server.js

```diff
@@ -236,6 +238,60 @@ app.delete("/services/:serviceName", function (req, res) {
     });
 });
 
+app.patch("/scopes", function (req, res) {
+  console.log(req.body);
+  req.body = JSON.parse(req.body);
+  let service, url;
+  let env = getEnv(req);
+  if (req.body != undefined && req.body.hasOwnProperty("scope")) {
+    service = req.body.scopes;
+  } else {
+    return res.status(400).send("scope key is missing");
+  }
+  if (req.body != undefined && req.body.hasOwnProperty("url")) {
+    url = req.body.url;
+  } else {
+    return res.status(400).send("url key is missing");
+  }
+
+  if (checkUrlUnsafe(url)) {
+    return res.status(400).send({
+      error: `URL is not trusted (${url})`,
+    });
+  }
+
+  verifyValidUrl(req, url)
+    .then(() => {
+      modify
+        .modifyScope(env, service, url)
+        .then((json) => {
+          res.send(json);
+        })
+        .catch((ex) => {
+          console.error(ex);
+          res
+            .status(500)
+            .send(`Could not write manifest file -- ${ex.toString()}`);
+        });
+    })
+    .catch((err) => {
+      res.status(400).send(err.message);
+    });
+});
+
+app.delete("/scopes/:scopeName", function (req, res) {
```

Comment:
Hmm can this API be part of PATCH /import-map.json? PATCH requests usually do not delete omitted properties, so maybe we can not. In that case, it might be okay to have this API.